### PR TITLE
Int guide migration

### PR DESCRIPTION
**Pull request by ryan-lgtm**

### CODE REVIEW

File: source/integrations/guides/bigcommerce.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+toc: {
+  "How to Connect Your BigCommerce Store": "#section-how-to-connect-your-bigcommerce-store",
+  "Calculate how much sales tax you've collected": "#section-calculate-how-much-sales-tax-you-have-collected"
+}
+---
+
+This guide will take you step by step through how to integrate TaxJar into your BigCommerce account. You’ll learn how to configure BigCommerce to automate sales tax calculations as well as what you need to do when it comes time to file.
+
+Sales tax is complex, with regulations changing constantly. For a primer on the basics of sales tax, including [nexus](https://www.taxjar.com/resources/sales-tax/nexus), [registration](https://www.taxjar.com/resources/sales-tax/registration), filing, reporting, calculations and more, please visit [Sales Tax Fundamentals](https://www.taxjar.com/resources/sales-tax). Also, be sure to take a look at our [Resource Center](https://www.taxjar.com/resources/), with articles, webinars and videos for beginners and experts alike.
```
Comment (dbernheisel) [inline source/integrations/guides/bigcommerce.md:22]: broken registration link.

File: source/integrations/guides/bigcommerce.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+You'll learn how TaxJar provides calculations and syncs invoices, cash sales, and credit memos for reporting / filing. Along the way, we'll configure your NetSuite account to collect sales tax where you have nexus, handle product / customer exemptions, and much more. For a primer on everything sales tax, read our [Sales Tax 101 guides](https://www.taxjar.com/learn-sales-tax/) before getting started.
```
Comment (dbernheisel) [inline source/integrations/guides/bigcommerce.md:24]: 101 guide link is broken.

File: source/integrations/guides/acumatica.md [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+  "Frequently Asked Questions": "#section-frequently-asked-questions",
+  "Integration Changelog": "#section-integration-changelog"
+}
+extra: "<div class='text-center'><a href='https://www.acumatica.com/acumatica-marketplace/taxjar-sales-tax-automation/' target='_blank'><img src='/images/guides/integrations/acumatica/acumatica_certified_application_retina.png' width='160'></a></div>"
+---
+
+This guide will take you step by step through how to integrate TaxJar with your Acumatica account. You’ll learn about calculations, invoicing syncs, reporting, filing, product and customer exemptions and more.
+
+Sales tax is complex, with regulations changing constantly. For a primer on the basics of sales tax, including [nexus](https://www.taxjar.com/resources/sales-tax/nexus), [registration](https://www.taxjar.com/resources/sales-tax/registration), filing, reporting, calculations and more, please visit [Sales Tax Fundamentals](https://www.taxjar.com/resources/sales-tax). Also, be sure to take a look at our [Resource Center](https://www.taxjar.com/resources/), with articles, webinars and videos for beginners and experts alike.
```
Comment (dbernheisel) [inline source/integrations/guides/acumatica.md:27]: broken registration link

File: source/integrations/guides/amazon.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+
+}
+resources: {
+  "Amazon Sales Tax Page": "https://www.amazon.com/gp/help/customer/display.html?nodeId=468512"
+}
+---
+
+This guide will take you step by step through how to integrate TaxJar with your Amazon account. With your TaxJar integration, you’ll enable real-time sales tax calculations and reporting from all of your channels -- along with the ability to automate your filing.
+
+Sales tax is complex, with regulations changing constantly. For a primer on the basics of sales tax, including [nexus](https://www.taxjar.com/resources/sales-tax/nexus), [registration](https://www.taxjar.com/resources/sales-tax/registration), filing, reporting, calculations and more, please visit [Sales Tax Fundamentals](https://www.taxjar.com/resources/sales-tax). Also, be sure to take a look at our [Resource Center](https://www.taxjar.com/resources/), with articles, webinars and videos for beginners and experts alike.
```
Comment (dbernheisel) [inline source/integrations/guides/amazon.md:28]: broken registration link

File: source/integrations/guides/amazon.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+
+Unfortunately for FBA sellers, the states don’t agree if shipping, handling, or gift wrap charges that are separately stated on an invoice are taxable or not. Some states say they are taxable, some say they’re not.
+
+Amazon allows you to decide if you want to tax shipping, handling, and gift wrap charges. The gift wrap portion of this is related to the actual service of gift-wrapping, not the paper that you wrap in item in.
+
+Here’s the way it works: if you choose to tax shipping, handling, or gift wrap then Amazon will charge sales tax at the same rate the item that’s being shipped or handles is being taxed. For example, if the sales tax rate for the item you’re selling is 8.25%, then the sales tax rate for shipping on that item will also be 8.25%. If the item is not taxable, then the shipping, handling, or gift wrap will not be taxed.
+
+## Tax Settings for FBA States
+
+<a class="w-inline-block expand-image" href="/images/guides/amazon/fba-sales-tax-chart.png" target="_blank"><img src="/images/guides/amazon/fba-sales-tax-chart.png"/></a>
```
Comment (dbernheisel) [inline source/integrations/guides/amazon.md:242]: broken image

File: source/integrations/guides/ebay.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+<!-- Use the table below to find your nexus states and fill in your sales tax rates.
+
+## eBay Sales Tax Table -->
+
+<!-- table goes here -->
+
+## How to Connect your eBay Store
+
+To connect TaxJar and eBay, your first step is to set up sales tax collection in your eBay store. We have a [step-by-step guide to setting up eBay sales tax collection here](https://blog.taxjar.com/sales-tax-on-ebay/).
```
Comment (dbernheisel) [inline source/integrations/guides/ebay.md:58]: this seems to redirect to the old URL for the guide you're migrating

File: source/integrations/guides/ebay.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+  "eBay Sales Tax Table": "#section-ebay-sales-tax-table",
+  "How to Connect your eBay Store": "#section-how-to-connect-your-ebay-store"
+}
+resources: {
+  "eBay Sales Tax Rates": "/ebay-tax-rates/"
+}
+---
+This guide will provide recommendations for collecting sales tax on eBay and take you step by step through how to integrate TaxJar with your eBay account. Your TaxJar integration will enable real-time sales tax calculations and reporting -- along with the ability to automate your filing.
+
+Sales tax is complex, with regulations changing constantly. For a primer on the basics of sales tax, including [nexus](https://www.taxjar.com/resources/sales-tax/nexus), [registration](https://www.taxjar.com/resources/sales-tax/registration), filing, reporting, calculations and more, please visit [Sales Tax Fundamentals](https://www.taxjar.com/resources/sales-tax). Also, be sure to take a look at our [Resource Center](https://www.taxjar.com/resources/), with articles, webinars and videos for beginners and experts alike.
```
Comment (dbernheisel) [inline source/integrations/guides/ebay.md:24]: registration link is broken. pretty much on all the guides.

File: source/integrations/guides/magento.md [hunk 8]
```diff
@@ -0,0 +1,527 @@
...
+That's it!
+
+If you're already logged in to your Magento admin panel, log out and log back in again for a clean slate.
+
+### Pre-Import Checklist
+
+Before going further, let's review some of your existing Magento settings to make sure TaxJar is clear for landing. Log in to your Magento admin panel.
+
+<div class="panel-group" id="pre-install-accordion" role="tablist" aria-multiselectable="true">
+  <div class="panel panel-default">
```
Comment (dbernheisel) [inline source/integrations/guides/magento.md:58]: image
styling is strange here throughout the guide, same on the magento 2 guide.

File: source/integrations/guides/paypal.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+Sales tax is complex, with regulations changing constantly. For a primer on the basics of sales tax, including [nexus](https://www.taxjar.com/resources/sales-tax/nexus), [registration](https://www.taxjar.com/resources/sales-tax/registration), filing, reporting, calculations and more, please visit [Sales Tax Fundamentals](https://www.taxjar.com/resources/sales-tax). Also, be sure to take a look at our [Resource Center](https://www.taxjar.com/resources/), with articles, webinars and videos for beginners and experts alike.
+
+Please note that TaxJar for PayPal requires a [TaxJar Professional or Premium](https://www.taxjar.com/how-it-works/) subscription. To sign up, or to upgrade your existing account, please [contact](https://www.taxjar.com/contact/) our sales team.
+
+## How to Connect Your PayPal Store
+
+To connect TaxJar and PayPal, your first step is to set up sales tax collection in your PayPal store.
+
+1. Login to your PayPal account and click here to visit [your PayPal Sales Tax page](https://www.paypal.com/webscr?cmd=_profile-sales-tax/)
```
Comment (dbernheisel) [inline source/integrations/guides/paypal.md:31]: this paypal link is broken.

File: source/integrations/guides/shopify.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+
+## How to Connect Your Shopify Store
+
+Shopify's built-in tax engine automatically collects sales tax for you, from wherever you tell it to - even if you have sales tax nexus in more than one state.
+
+To turn on sales tax collection in Shopify, simply go to Settings > Taxes. 
+
+The Shopify Tax Manual quickly and thoroughly walks you through setting up sales tax.
+
+<div class="panel panel-success">
```
Comment (dbernheisel) [inline source/integrations/guides/shopify.md:36]: styling here is not quite right. Feels like the check icon should be left-aligned.
image

**Review by fastdivision — APPROVED** [review]

**Pull request merged**